app/controllers/rebate_forms_controller.rb

```diff
@@ -4,25 +4,23 @@ class RebateFormsController < ApiController
   jsonapi resource: RebateFormResource
   strong_resource :rebate_form
 
-  def show
-    scope = jsonapi_scope(RebateForm.where(token: params[:id]))
-    instance = scope.resolve.first
-    raise JsonapiCompliable::Errors::RecordNotFound unless instance
+  def show_by_jwt
+    token = params[:jwt]
 
-    render_jsonapi(instance, scope: false)
-  end
+    raise JsonapiCompliable::Errors::RecordNotFound unless token
 
-  def create
-    rebate_form = RebateFormsService.new(rebate_form_params).update
+    decoded_token = decode_jwt(token)
 
-    if rebate_form.errors.any?
-      render_errors_for(rebate_form)
-    else
-      render_jsonapi(rebate_form, scope: false)
-    end
+    rebate_form_id = decoded_token[0]['rebate_form_id']
```

Comment:
Minor issue; you should check the value of `decoded_token.per` to be equal to `fetch_application_and_submit_signatures`. This will prevent executing actions with a valid JWT token but not generated for a given action.